pkg/systemd/logs.html

```diff
@@ -51,11 +51,15 @@
           <span class="caret"></span>
         </button>
         <ul id="prio-lists" class="dropdown-menu" role="menu">
-          <li><a data-prio="*" translatable="yes">Everything</a></li>
-          <li><a data-prio="5" translatable="yes">Problems, Errors, Warnings, Notices</a></li>
-          <li><a data-prio="4" translatable="yes">Problems, Errors, Warnings</a></li>
-          <li><a data-prio="3" translatable="yes">Problems, Errors</a></li>
-          <li><a data-prio="2" translatable="yes">Only Problems</a></li>
+          <li><a data-prio="*" translatable="yes">All</a></li>
+          <li><a data-prio="0" translatable="yes">Emerg</a></li>
+          <li><a data-prio="1" translatable="yes">Alert</a></li>
+          <li><a data-prio="2" translatable="yes">Crit</a></li>
+          <li><a data-prio="3" translatable="yes">Err</a></li>
```

Comment:
I'm not too happy about introducing these rather arbitrary abbreviations. There is no reason for that kind of brevity in the UI, and it's not clear how to translate them either. Maybe the explanations in `man syslog` would be a better basis for the strings here?

Could this keep "Everything" instead of "All" to avoid breaking all existing translations?

In the current version, repeating the severity levels and appending new ones is certainly bad too, and it's also not clear what the difference between "problem" and "error" is. I think something outside of the dropdown should point out that this applies to "this severity level or higher".

@andreasn,  @garrett, do you have some ideas here?

Reply:
Patternfly has two important rules for abbreviations. One is to write only those that users are familiar with, and the other one is to use the consistently.
https://www.patternfly.org/styles/terminology-and-wording/#abbreviations

Our target audience of junior Linux admins and former Windows admins are not familiar with the Kernel log level abbreviations, so it's better to write them out in full.
It's also breaking the second rule, because "Warning" is written in full, but "Err" and "Crit" is not, so that's inconsistent.

Reply:
Thanks for feedback! I'll provide required changes.